src/TestFramework/PhpUnit/Config/XmlConfigurationManipulator.php

```diff
@@ -82,9 +82,11 @@ public function replaceWithAbsolutePaths(SafeDOMXPath $xPath): void
     public function removeExistingLoggers(SafeDOMXPath $xPath): void
     {
         foreach ($xPath->query('/phpunit/logging') as $node) {
-            $document = $xPath->document->documentElement;
-            Assert::isInstanceOf($document, DOMElement::class);
-            $document->removeChild($node);
+            $node->parentNode->removeChild($node);
```

Comment:
what do you think about adding a comment to this method (or method body) than loggers will be added by Infectin through CLI arguments? will be easier to understand the whole picture for future readers

Reply:
How about now?

Reply:
cool!